webpack.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+    mode: "none",
```

Comment:
Question: Do we want to include devtool: "source-map" (or another variant suggested for production builds, see [webcompat docs here](https://webpack.js.org/configuration/devtool/#production)) to ensure webcompat is going to build a source map file?

Reply:
If it's easy to include, then let's. Otherwise, OK not to bother.